plugins/reposync.py

```diff
@@ -148,7 +150,7 @@ def run(self):
                 self.delete_old_local_packages(repo, pkglist)
 
     def repo_target(self, repo):
-        return _pkgdir(self.opts.destdir or self.opts.download_path, repo.id)
+        return _pkgdir(self.opts.destdir or self.opts.download_path, repo.id if not self.opts.omit_repoid else '')
```

Comment:
please split this line into two to keep line length under 100 charactere.

Reply:
I've splitted the line